src/controllers/login.js

```diff
@@ -0,0 +1,30 @@
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+require('env2')('.env');
+
+exports.post = (req, res) => {
+  const { username, password } = req.body;
+  const hashed = process.env.password;
+  const userhere = process.env.username;
+  const secret = process.env.SECRET;
+  if (userhere === username) {
```

Comment:
since you are using deconstructering in line 6, why not also use it for line 7 to 9 as const `{ hashed , secret , username:userhere} = process.env`